front/src/app/projects/projects.component.html

```diff
@@ -1,61 +1,172 @@
+                            </div>
+                        </span>
+                    </div>
+                </div>
+            </mat-grid-tile>
+            <mat-grid-tile *ngFor="let project of projects | async; index as idx">
+                <div class="w-100 h-100 p-2">
+                    <div class="tile item w-100 h-100">
+                        <div class="row">
+                            <div class="col-12 icon">&nbsp;
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-12 text-center">
+                                <h5><b>{{project.name}}</b></h5>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-12" style="text-align: center;"><b>
+                                    <span *ngIf="project.status=='positive'" style="font-size: 1em; color: green">
+                                        <i class="fa fa-check-square"></i>&nbsp;All successful
+                                    </span>
+                                    <span *ngIf="project.status=='negative'" style="font-size: 1em; color: red">
+                                        <i class="fa fa-minus-square"></i>&nbsp;At least one negative
+                                    </span>
+                                    <span *ngIf="project.status=='waiting'" style="font-size: 1em; color: orange">
+                                        <i class="fa fa-pause-circle"></i>&nbsp;At least one waiting
+                                    </span></b>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-1"></div>
+                            <div class="col-10">
+                                <div class="tool-scrollbar">
+                                    <table class="table table-hover table-borderless">
+                                        <tbody *ngFor="let scan of project.scans; index as i;">
+                                            <tr>
+                                                <th scope="row">{{i+1}}</th>
+                                                <td>{{scan.toolName}}</td>
+                                                <td>{{scan.stringDate}}</td>
+                                            </tr>
+                                        </tbody>
+                                    </table>
                                 </div>
                             </div>
                         </div>
+                        <div class="row">
+                            <div class="col-12 text-center align-self-end mt-2">
+                                <span [routerLink]="['/project', project.name]">
+                                    <button type="button" class="btn btn-outline-dark" (click)="set(project)">
```

Comment:
Skoro ścieżka zawiera nazwę projektu, to ustawianie wybranego projektu w stanie zamiast pobrania sobie po nazwie ze ścieżki go później ze stanu jest drogą naokoło. Jeśli nazwa projektu nie jest unikalna, to można zamiast nazwy użyć id projektu.